src/dropbox.js

```diff
@@ -811,7 +837,7 @@
           return Promise.reject(new Error("Cannot delete '" + path + "': too many files involved"));
         }
 
-        if (resp.status === 200) {
+        if (resp.status === 200 || resp.status === 404) {
```

Comment:
It's a delete request, so if it tries to delete something that doesn't exist, that would result in a 404 I think.